ON-CALL.md

```diff
@@ -0,0 +1,44 @@
+# Istio On-call Playbook
+
+## Who
+Every week, one Istio developer is on-call and is responsible for maintaining Istio build and for helping out users and other developers. The on-call person should prioritize on-call duties on top of the daily work.
+
+## Schedule
+The on-call schedule can be found [here](https://docs.google.com/spreadsheets/d/1FaHwPpad3F3hva2suJweNeTnocjKtLnbgLkyMRPzgUY/edit#gid=1475801904).
+
+On-call duty starts on Tuesday at noon PST, ends the following week on Tuesday at noon PST, and is performed during regular working hours for your time zone.
+
+## Prerequisites
+* Follow the [Istio dev guide](https://github.com/istio/istio/blob/master/DEV-GUIDE.md) to set up your environment
+* Join google group [istio-users](https://groups.google.com/forum/#!forum/istio-users)
+* Join google groups [istio-oncall](https://groups.google.com/forum/#!forum/istio-oncall)
+* Join the `oncall` Slack channel
+
+## Responsibilities
+* Build cop: monitor the builds, the presubmit automated tests, the postsubmit automated tests:
+  * Familiarize yourself with the current [open issues affecting automated tests](https://github.com/istio/istio/issues?q=is%3Aopen+is%3Aissue+label%3Akind%2Ftest-failure).
```

Comment:
That seems a too long list to be familiar with.  Is there a shortened list just for build/critical test break issue?

Reply:
This is the list unfortunately, 20 open issues. If it is too long, people should prioritize fixing those bugs.

Reply:
This is the list for critical issues, not all the issues.